_posts/2023-08-04-deploying-OIDC-webapp-on-openshift.adoc

```diff
@@ -0,0 +1,161 @@
+
+== Setting up your environment
+For this walkthrough, will be making use of the https://github.com/PrarthonaPaul/elytron-examples/tree/elytron-oidc-client-scope/elytron-oidc-client-scope[elytron-oidc-client-scope] project which makes use of the ability to configure additional scope values for OpenID Connect. First, clone the `elytron-example` repo locally and navigate to the `elytron-oidc-client-scope` project:
+```
+git clone https://github.com/wildfly-security-incubator/elytron-examples
+cd elytron-examples/elytron-oidc-client-scope
+```
+We can deploy this application using Red Hat OpenShift https://developers.redhat.com/developer-sandbox[Developer Sandbox]. There are other https://www.redhat.com/en/technologies/cloud-computing/openshift/try-it[options] available for running OpenShift. Please refer to this https://docs.wildfly.org/29/Getting_Started_on_OpenShift.html[getting started guide] to learn more about WildFly on OpenShift. We will be making use of the following tools for this walkthrough: 
+
+* https://docs.docker.com/[Docker] (you can also use Podman)  
+* https://developers.redhat.com/developer-sandbox[Developer Sandbox]
+* https://docs.openshift.com/container-platform/4.11/cli_reference/openshift_cli/getting-started-cli.html[OpenShift CLI]
+* https://quay.io/repository/[Quay]
+* https://helm.sh/[Helm]
+
+=== Logging into OpenShift CLI
+
+If you are not logged into OpenShift on your cli, navigate to Developer Sandbox, click on your username at the top right corner, copy your login command and paste it on your terminal. 
+
+Once you have your environment set up with the required tools, we can move on to the next step to build and deploy our application on OpenShift. 
+
+== Setting up Red Hat Single Sign On 
+Developer Sandbox gives us access to the enterprise version of Red Hat's OpenID provider, Red Hat Single Sign On (RH-SSO). On the topology view of OpenShift, click on the `Add to Project` button on the top left and add the version of RH-SSO that is compatible with the current version of OpehShift you are running and wait for it to finish deploying. Please refer to the https://access.redhat.com/documentation/en-us/red_hat_single_sign-on/7.2/html/red_hat_single_sign-on_for_openshift/introduction[RH-SSO for OpenShift Documentation] to learn more about it.
+
+Next click on the `Open URL` button on the pod to be redirected to the RH-SSO landing page and click o `Administration Console`. You will be able to sign in using the admin username and password that is specified in the yaml configuration of the RH-SSO pod. 
+
+From here, create a realm called `myrealm`, and a client called `myclient`. 
+
+After registering our client, `myclient`, we also need to configure `valid redirect URIs`. Simply click on `Clients` and then on `myclient`. In the `Valid Redirect URIs` field, enter `*`.
```

Comment:
I think we should be able to specify APP_URL/* for Valid Redirect URIs

Reply:
Fixed!
added instructions to change the link from `https` to `http` for `redirect uri`